src/components/Asset/AssetActions/TermsAndConditionsCheckbox.tsx

```diff
@@ -0,0 +1,25 @@
+import Input from '@components/@shared/FormInput'
+import { Field } from 'formik'
+
+export default function TermsAndConditionsCheckbox({
+  actions,
+  onChange,
+  disabled
+}: {
+  actions: any
+  onChange?: (termsAndConditions: boolean) => void
+  disabled: boolean
+}) {
+  return (
+    <Field
+      name="termsAndConditions"
+      type="checkbox"
+      options={['Terms and Conditions']}
+      prefixes={['I agree to the']}
+      actions={actions}
+      component={Input}
+      disabled={disabled}
+      onChange={onChange}
+    />
```

Comment:
suggestion(blocking): to follow the repository structure we should try to still keep this content in a `/content/some.json`